src/style.css

```
@@ -0,0 +1,133 @@
+* {
+  margin: 0;
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: #060b35;
+}
+
+.card {
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  background: #fff;
+  border-radius: 5px;
+}
+
+.card__monitor {
+  margin: 32px 19px 0;
+}
+
+.card__information {
+  margin: 35px 16px 16px;
+}
+
+.card__code {
+  margin-top: 4px;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+}
+
+.card__rating {
+  display: flex;
+  margin-top: 16px;
+  justify-content: space-between;
+}
+
+.card__stars {
+  display: flex;
+  align-items: flex-start;
+
+}
+
+.star {
+  display: flex;
+  align-items: center;
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
```

here and everywhere group modifiers and split by empty line
```suggestion
.star {
  display: flex;
  align-items: center;
  
  height: 16px;
  width: 16px;
  margin-right: 4px;
  
  background-image: url(./images/star.svg);
  background-repeat: no-repeat;
  background-position: center;
}
```
see [recommendations](http://codeguide.academy/html-css.html#css-order)